libr/bin/obj.c

```diff
@@ -148,6 +149,7 @@ R_IPI RBinObject *r_bin_object_new(RBinFile *binfile, RBinPlugin *plugin, ut64 b
 	}
 	o->obj_size = bytes && (bytes_sz >= sz + offset)? sz: 0;
 	o->boffset = offset;
+	o->strings_db = ht_new (NULL, NULL, NULL);
```

Comment:
nope! please provide a `free_kv` function that does this:
```C
static void free_kv(HtKv *kv) {
     free (kv->key);
}
```

by default the ht strdups the key, so you need to free it. we should probably have a default for this, i'm going to work on it but in the meantime please implement it. there are other places where this is already done and i will replace them once i'll change things in sdb.

Reply:
done